CHANGES.txt

```diff
@@ -1,18 +1,149 @@
 Enable CHANGELOG
 ================
 
-Changes since Enable 4.8.0
-==========================
+Enable 5.0.0
+============
 
 Enhancements
 ------------
 
+* Contribute examples to ``etsdemo``. (#494)
+* Add ``Container.components`` setter. (#387)
+
+Changes
+-------
+
+* Make kiva explorer a popup. (#510)
+* Remove import time side effect of creating the font cache in the
+  ``font_manager``. (#488)
+* Add opacity to null-toolkit-defined rgba_color_trait. (#374)
+
 Fixes
 -----
 
+* Allow unhandled key and mouse-wheel events to bubble up. (#552)
+* Fix ``Font.findfontname`` ``AttributeError`` when the default font manager
+  has not been initialized. (#531))
+* Fix errors in examples. (#508)
+* Fix ``AttributeError`` on Python 3.9 due to the use of removed API in
+  ``xml``. (#492)
+* Make font style- and stretch- check case insensitive. (#405)
+* Add a missing ``self`` argument to a method. (#468)
+* WxPython4 fixes. (#403)
+* Pass file object into ``TTCollection`` and close when done. (#378)
+* Fix ``DeprecationWarning``\s due to escape characters. (#371)
+* Don't mess with logging configuration. (#370)
+* Fix KeyError while parsing ttfFontProperty. (#365)
+
+Documentation
+-------------
+
+* Improve module docstrings in enable api modules. (#564)
+* Use templates when generating api docs for ``enable`` and ``kiva``. (#526)
+* Ignore more toolkit-specific modules in api documentation. (#527)
+* Dont document null submodule in api docs. (#522)
+* Add module docstrings to kiva and enable api modules. (#518)
+* Clean up a few demos. (#511)
+* Add a click command to build docs. (#499)
+* Fix failing documentation build on Python 3. (#478)
+* Expand ``DragTool`` comments. (#452)
+* Fix a number of issues with the examples. (#402)
+* Small fixes for examples. (#388)
+* Add a demo runner for enable. (#383)
+
+Testing
+-------
+
+* Clean up some of the agg tests. (#573)
+* Add a unit test for drawing images. (#572)
+* Add ``tox.ini`` for flake8 control. (#559)
+* Unskip most of the ``TestGLDrawing`` tests on OSX. (#540)
+* Address resource warnings because of open file handles. (#529)
+* Rename test module so it gets discovered by unittest. (#528)
+* Fix tests for PySide2 and add PySide2 to the testing matrix. (#484)
+* Make tests for font manager quieter. (#487)
+* Use unittest as the test runner instead of nose. (#475)
+* Switch on default warnings flag for CI test command. (#469)
```

Comment:
the distinction between some of the items in this section vs Build and CI is a little unclear to me, but its pretty inconsequential.  Not a suggested change